src/components/sortable_table/header.js

```diff
@@ -23,78 +22,77 @@ var TableHeader = React.createClass({
     onLabelClick: React.PropTypes.func
   },
 
-  getInitialState: function() {
-    /*
-     * Keeps track of sort direction on each column.
-     */
-    var directionHash = _.zipObject(this.props.columns,
-      _.times(this.props.columns.length, function() { return 'ascending'; }));
+  /*
+   * Keeps track of sort direction on each column.
+   */
+  getInitialState() {
+    let directionHash = {};
+    this.props.columns.forEach((column) => {
+      directionHash[column] = 'ascending';
+    });
 
     return {
-      directionHash: directionHash
+      directionHash
     };
   },
 
-  onLabelClick: function(columnName, ev) {
-    /*
-     * Grabs table type and sort option and passes to sort callback.
-     * Flips direction in direction hash.
-     */
-    var direction = this.state.directionHash[columnName] === 'ascending' ? 'descending' : 'ascending';
-    var hashCopy = _.cloneDeep(this.state.directionHash);
+  /*
+   * Grabs table type and sort option and passes to sort callback.
+   * Flips direction in direction hash.
+   */
+  onLabelClick(columnName, event) {
+    let direction = this.state.directionHash[columnName] === 'ascending' ? 'descending' : 'ascending';
+    let updatedHash = assign({}, this.state.directionHash, {
```

Comment:
Interesting. These are the babel things we're using at the current gig, but no polyfill. Must be included in one of the presets we're using. 

![image](https://cloud.githubusercontent.com/assets/3853/12333295/6c23e55e-bac1-11e5-8a59-8791e835f1e5.png)